examples/sample-triggers/sample-triggers.go

```diff
@@ -22,18 +22,18 @@ var OpPeople = hbot.Trigger{
 		}
 		return false
 	},
-	func(irc *hbot.Bot, mes *hbot.Message) bool {
+	Action: func(irc *hbot.Bot, mes *hbot.Message) bool {
```

Comment:
The convention in the rest of the codebase is to use either `m` or `msg` for message variable names. (`m` is preferred unless it collides with another name)

Reply:
I fixed that and settled for 'm'